config/management/src/storage.rs

```diff
@@ -13,37 +13,11 @@ use std::str::FromStr;
 
 /// A helper to handle common error handling and functionality for tooling
 pub struct StorageWrapper {
-    storage_name: &'static str,
-    storage: Storage,
+    pub(crate) storage_name: &'static str,
```

Comment:
Interesting, that's a new one for me. It's like I guess a Java `protected`.  We should probably be doing that lots of places in our code.

Reply:
ahh, well looking back now, I could have probably done this in such a way to use the new function.